features/sensor_readings_api/receive.feature

```diff
@@ -44,7 +52,11 @@ Feature: Receive Sensor Readings
     When I send a POST request to "/sensor_readings" with the following:
     """
     {
-      "sensor_name": "DOESN'T EXIST", "calibrated_value": 47, "uncalibrated_value": 11
+      "calibrated_value": 47,
+      "uncalibrated_value": 11,
+      "sensor": {
+        "name": "DOESN'T EXIST"
+      }
```

Comment:
@kabel42 @drjakob this is a *changed* cucumber feature. Note that other lookup attributes (```name``` in this case) are nested inside a ```"sensor": {.. }``` hash